### ISSUE TITLE
SDF or occupancy field

### ISSUE THREAD

**Issue by SeanChenxy** [zhengyuf/IMavatar#2]
Hi, Yufeng. Thanks for sharing codes. I find you use SDF in the code, but the paper says "reconstruct an occupancy field
instead of an SDF field." Did I understand something wrong? Looking for your reply.

**Comment by zhengyuf** [comment]
Hey,

We didn't call it an SDF because we didn't use the eikonal loss to ensure unit normal and didn't leverage sphere tracking. 
Yes, the predicted values from the geometry network are called sdf and they are unlimited in range like SDFs, but they can also be seen as the logits of an occupancy.
[Here](https://github.com/zhengyuf/IMavatar/blob/b5b7cf2f43bd482a3f355b3a21c0012e64326eae/code/model/loss.py#L58) we supervised the values with binary_cross_entropy_with_logits loss. 
So overall, we think we used can be more accurately described by an occupancy field.
